test/Tizen.NUI.Scene3D.Sample/Scene3DSample.cs

```diff
@@ -0,0 +1,245 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using Tizen.NUI;
+using Tizen.NUI.BaseComponents;
+using Tizen.NUI.Constants;
+using Tizen.NUI.Scene3D;
+using System.Collections.Generic;
+
+class Scene3DSample : NUIApplication
+{
+    static string IMAGE_DIR = Tizen.Applications.Application.Current.DirectoryInfo.Resource + "image/";
+    static string MODEL_DIR = Tizen.Applications.Application.Current.DirectoryInfo.Resource + "model/";
+
+    Window mWindow;
+    Vector2 mWindowSize;
+
+    SceneView mSceneView;
+    Model     mModel;
+
+    uint currentCameraIndex = 0u;
+    const int cameraAnimationDurationMilliSeconds = 2000; // milliSeconds
+
+    Animation mModelRotateAnimation;
+    const int modelRotateAnimationDurationMilliSeconds = 10000; // milliSeconds
+
+    protected void CreateSceneView()
+    {
+        mSceneView = new SceneView()
+        {
+            SizeWidth = mWindowSize.Width,
+            SizeHeight = mWindowSize.Height,
+            PositionX = 0.0f,
+            PositionY = 0.0f,
+            PivotPoint = PivotPoint.TopLeft,
+            ParentOrigin = ParentOrigin.TopLeft,
+            PositionUsesPivotPoint = true,
+        };
+
+        SetupSceneViewCamera(mSceneView);
+
+        mWindow.Add(mSceneView);
+    }
+    private void SetupSceneViewCamera(SceneView sceneView)
+    {
+        // Default camera setting
+        // Note : SceneView always have 1 default camera.
+        Tizen.NUI.Scene3D.Camera defaultCamera = sceneView.GetCamera(0u);
+
+        defaultCamera.PositionX = 0.0f;
+        defaultCamera.PositionY = 0.0f;
+        defaultCamera.PositionZ = 5.55f;
```

Comment:
Later, these specific numbers can be defined on the above together.
Like animation duration milliseconds.

Reply:
Done.